resources/js/Components/ItemIdSearchTextarea.vue

```diff
@@ -98,7 +98,8 @@ defineExpose({validate, serializeInput, validationError});
 </script>
 
 <style lang="scss">
-@import '~@wmde/wikit-tokens/dist/_variables.scss';
+/* stylelint-disable-next-line scss/load-no-partial-leading-underscore */
+@import '~@wmde/wikit-tokens/dist/_variables';
```

Comment:
oops, i think we can remove this. but out of the scope of this task

Reply:
Yeah, I don't think this is part of this task. Is this a leftover from the tokens?

Reply:
yeah i guess so. i'll do it as part of the grid system